sdk/eventhubs/azure-messaging-eventhubs/pom.xml

```diff
@@ -46,6 +46,12 @@
     </dependency>
 
     <!-- Test dependencies -->
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
```

Comment:
Prefer to take the released version dependency instead of `beta`

Reply:
I copied it from storage. . . Should we be using the released one there too?
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob/pom.xml#L92-L97